.codecov.yml

```diff
@@ -0,0 +1,19 @@
+coverage:
```

Comment:
Generally, yes. #18 was kind of a problem since it encompassed both the skeleton project creation and implementation of features. The story really should have been split, but since it wasn't, you'll see both things here.